packages/main/src/DayPicker.js

```diff
@@ -863,19 +860,7 @@ class DayPicker extends UI5Element {
 	}
 
 	_isOutOfSelectableRange(date) {
-		const currentDate = date._oUDate ? date.toLocalJSDate() : CalendarDate.fromTimestamp(date).toLocalJSDate();
-		const minDate = this._minDateObject;
-		const maxDate = this._maxDateObject;
-
-		currentDate.setHours(0);
-		if (minDate) {
-			minDate.setHours(0);
-		}
-		if (maxDate) {
-			maxDate.setHours(0);
-		}
-
-		return currentDate > maxDate || currentDate < minDate;
+		return date.valueOf() < this._minDate || date.valueOf() > this._maxDate;
```

Comment:
I am not against this simplification, but just to confirm: 
Is it possible for "this._minDate" or  "this._maxDate" to be undefined and would the check still work in such case.

Reply:
Currently it is not possible for those values to be undefined. There are default min and max values if users haven't set such as attributes. The default values would be from 0001.0.1 as minimum date to 9999.11.31 as maximum date. Those dates are also properly converted for the different calendar types.

Reply:
It's not possible based on this implementation because the getters _min/maxDate are requesting the min/max dates set by the developer and if there is not a explicitly set min or max date, it calls the _getMaxCalendarDate(Or min) which are the 1st calendar date(1/1/0001) or the last possible date in the calendar(31/12/9999) supported.

Reply:
Ok, thanks for the clarification, I approve the PR then.